cypress/integration/index.test.js

```diff
@@ -0,0 +1,31 @@
+describe("Index", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:3000/");
+  });
+
+  it("displays a top bar", () => {
+    cy.contains("Go Fund Me");
+    cy.contains("Help me go on a vacation");
+  })
+
+  it("displays flavor text", () => {
```

Comment:
what is flavor text?

Reply:
I forget not everybody used to be _really_ into Magic: The Gathering...